tensorflow/core/lib/io/inputbuffer.cc

```diff
@@ -85,7 +85,7 @@ Status InputBuffer::ReadNBytes(int64 bytes_to_read, string* result) {
   result->resize(bytes_to_read);
   size_t bytes_read = 0;
   Status status = ReadNBytes(bytes_to_read, &(*result)[0], &bytes_read);
```

Comment:
Would be better to replace the signature of ReadNBytes to not have `size_t` as the type of the last argument. Let's try to do this in another PR (as there would be multiple places to change)

Reply:
Ok, what I'll do is to revert all the files that produce warnings related ReadNBytes(), and as you recommend address them in another PR.